tests/structure/test_notears.py

```diff
@@ -573,3 +599,62 @@ def test_f1_score(self, train_data_idx, train_model_idx):
         f1_score = 2 * (precision * recall) / (precision + recall)
 
         assert f1_score > 0.8
+
+    def test_f1score_generated(self, adjacency_mat_num_stability):
+        """Structure learnt from regularisation should have very high f1 score relative to the ground truth"""
+        df = pd.DataFrame(
+            10 * adjacency_mat_num_stability,
+            columns=["a", "b", "c", "d", "e"],
+            index=["a", "b", "c", "d", "e"],
+        )
+        train_model = StructureModel(df.values)
+        X = generate_continuous_dataframe(
+            StructureModel(df), 50, noise_scale=1, seed=20
+        )
+        g = from_numpy_lasso(
+            X.loc[:, ["a", "b", "c", "d", "e"]].values, 0.1, w_threshold=0.1
```

Comment:
Equivalently, you can do this without loc
```suggestion
            X[["a", "b", "c", "d", "e"]].values, 0.1, w_threshold=0.1
```